windows-apps-src/design/style/reveal.md

```diff
@@ -134,15 +134,23 @@ These effects are defined by two brushes:
 In most cases we handle the usage of both of them by turning Reveal on automatically for a certain controls. However, other controls will need to be enabled through applying a style, or changing their templates directly.
 
 ### When to add Reveal
-You can add Reveal to your custom controls--but before you do, consider the type of control and how it behaves. 
-* If your custom control is a single interactive element and doesn't have similar controls sharing it's space (such as menu items in a menu), it's likely that your custom control doesn't need Reveal.  
-* If you have a grouping of related interactive content or elements, then it's likely that that region of your app does need Reveal - this is commonly referred to as a [Commanding](https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/collection-commanding) surface.
+You can add Reveal to your custom controls--but before you do, consider the type of control and how it behaves.
 
-For example, a button by itself shouldn't use reveal, but a set of buttons in a command bar should use Reveal.
+Add Reveal when
+* **Your control is in a group of other similarly-sized, interactive items**
+Enable Reveal on all of the controls in the group. The Reveal effect will make it easier for users to see which specific control they're interacting with.
+* **Your custom control part of a [Commanding Collection](https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/collection-commanding)?**
+Reveal is a great choice for situations like these.
+
+Don't add Reveal when
+* **Your control isn't near hero content or is far away from other interactive elements**
+Reveal is not the best choice for these scenarios.
+
+In general, Reveal highlight brings attention to certain areas of your app, or to pieces of interactive UI that you want most of your user's focus to be drawn to. Adding Reveal to secondary UI can cause distraction, or mislead their attention; however, adding it to primary UI or hero moments in your app is where Reveal really shines.
```

Comment:
>Adding Reveal to secondary UI can cause distraction, or mislead their attention; however, adding it to primary UI or hero moments in your app is where Reveal really shines.

I do not understand what you mean by primary/secondary UI. What is hero moments?

Reply:
Examples of "hero moments" of UI are things like the large image on the store front page when you first open the app, or the buttons in the calculator app, or the ribbon in word/office product. Any piece of UI that's crucial to the overall experience of your app, that when the user is there you want them to have a good experience using. Typically those areas are also UI dense - which is where Reveal is most useful.

Reply:
Do we want reveal on the hero images? That was not expected...

How about something like
"In order to bring the appropriate attention, do not add reveal on secondary actions that are not priority for your applications. However, adding it to where many action items are present at a time (commandbar, calculator's buttons, etc.) is a great use of reveal."